python/cuml/test/dask/conftest.py

```diff
@@ -11,9 +11,12 @@
 @pytest.fixture(scope="module")
 def cluster():
 
+    print("Starting cluster")
```

Comment:
Stray debugging code?

Reply:
I am actually hoping we can keep this in there. It's allowing us to see when these cancelled futures are happening. For example, sometimes they are happening in `cluster.close()`, which is pretty benign. When they happen before stopping the cluster, it's an indication of an issue. It doesn't seem to hurt much by keepig them in there because they only print out once per (Dask) test file currently.